### PR TITLE
Fix Directory Results Formatting

### LINKED ISSUE #167: Reset Button on Directory Breaks/Fails to Redirect
Problem caused by #152 where the link on the reset button on directory page is not changed, so button onclick leads to a page not found ("/search").

### PR DESCRIPTION
**Pull request by grinnelldennis**
Resolves #167 , #154 

# Changes

- fix link on reset button click in `directory.html.erb`(#167)
- format reset button to look like search button, `directory.html.erb` and `custom.scss`
- fix formatting when searching for residents from a specific neighborhood over 98 results in `directory.html.erb` (#154)

# Testing

0. Run `db rake:dummy_seed` if your local database is not already populated. The command can be used in succession to create distinct profiles in the database.  
1. Login and navigate to the Directory page,
2. Search by neighborhood,
4. Click reset button->search results should reset. 

**Require >98 search results to confirm the problem is resolved, but search results should now display correctly regardless of number of results**

### CODE REVIEW

**Comment by grinnelldennis** [comment]
MERGE ONLY AFTER #164

File: app/views/static_pages/directory.html.erb [hunk 1]
```diff
@@ -32,11 +32,8 @@
         </div>
       </div>
       <br/><center>
```
Comment (AlexanderOtavka) [inline app/views/static_pages/directory.html.erb]: The `<center>` element is deprecated.  You should use a `<nav>`, and style it

File: app/views/static_pages/directory.html.erb [hunk 2]
```diff
@@ -49,7 +46,7 @@
       <div class="two">No results. Try searching for something else.</div>
      <% else %>
       <% @results.each do |person| %>
-       <p> <font size="5">
+       <p style="font-size: x-large">
```
Comment (AlexanderOtavka) [inline app/views/static_pages/directory.html.erb]: This is an improvement, but we should probably use a stylesheet for this font size, and use [rem units](https://www.sitepoint.com/understanding-and-using-rem-units-in-css/).

File: app/views/static_pages/directory.html.erb [hunk 3]
```diff
@@ -49,7 +46,7 @@
       <div class="two">No results. Try searching for something else.</div>
      <% else %>
       <% @results.each do |person| %>
-       <p> <font size="5">
+       <p style="font-size: x-large"> 
         <a href= <%= profile_path(person.id) %> >   <%= person.first_name %> <%= person.last_name %></a> 
        <font/></p>
```
Comment (AlexanderOtavka) [inline app/views/static_pages/directory.html.erb]: You forgot to delete the `<font/>`, which is invalid HTML anyway.

**Pull request merged**